target/linux/realtek/dts-5.10/rtl8393_d-link_dgs-1210-52.dts

```diff
@@ -0,0 +1,371 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "rtl839x.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+
+#define COMBO_SFP_PHY(n, s) \
+	phy##n: ethernet-phy@##n { \
+		compatible = "ethernet-phy-ieee802.3-c22"; \
+		sfp = <&sfp##s>; \
+		reg = <##n>; \
+	};
+
+#define COMBO_PORT(n, s, m) \
+	port@##n { \
+		reg = <##n>; \
+		label = SWITCH_PORT_LABEL(s) ; \
+		phy-handle = <&phy##n>; \
+		phy-mode = #m ; \
+	};
```

Comment:
Perhaps these macros should be moved to a common include for the realtek target, so they don't have to be defined for every device/subtarget.

Reply:
I don't know if I should create them based on a device where it does not work (I believe because the driver requires adaptations). Once DGS-1210-16 SFP ports support is submitted and merged, it would be more legit to add that new macro file for others to import, incluiding this one.

I just included those macros to keep the SFP information I have already gathered anchored to a valid DTS file.